apps/src/p5lab/P5Lab.js

```diff
@@ -440,7 +440,10 @@ P5Lab.prototype.init = function(config) {
     config.initialAnimationList && !config.embed && !config.hasContainedLevels
       ? config.initialAnimationList
       : this.startAnimations;
-  getStore().dispatch(setInitialAnimationList(initialAnimationList));
+
+  getStore().dispatch(
+    setInitialAnimationList(initialAnimationList, this.isSpritelab)
```

Comment:
nit: can you add a comment for the boolean parameter `this.isSpritelab /* shouldRunV3Migration */`?